content-engine/tutorial/models/delete.mdx

```diff
@@ -0,0 +1,32 @@
+---
+slug: /content-engine/tutorial/models/delete
+title: Content Engine Models - Delete
+description: Learn how to delete a Model in Content Engine.
+---
+
+**Note**: These tutorials assume that you have already [installed the plugin](../../introduction.mdx).
+
+## Delete a Model
+
+A model is a container that holds fields for use in content generation in publisher.
+
+### Steps to Delete a Model
+---
+1. Click on Content Modeler in the WordPress side menu
+
+    ![Content Engine sync](../../images/acm-sidebar.png)
+
+1. If it's not on Models, click Models under Content Modeler in the WordPress side menu
+
+1. You will see the current models listed. Click the triple dot menu and select Delete from the popup menu
+
+    ![Content Engine sync](../../images/delete-model.png)
+
+1. Confirm deletion by clicking Delete in the modal that pops up. Click Cancel to keep the model
+    
+    ![Content Engine sync](../../images/Screen_Shot_2021-09-01_at_12.08.32_PM.png)
+
+## Feedback
+---
+- **Questions?** Email us ce-beta@wpengine.com
+- **Feedback**: Your feedback helps shape the future of Content Engine. To provide feedback, fill out our [Content Engine Feedback Form](https://docs.google.com/forms/d/e/1FAIpQLSecvuZ_EMiTIOlTSwcW1JnPQcFbAcCOwGlhURkzBI8Ps9vFzA/viewform).
```

Comment:
Do we have one of these forms for ACM? When I follow the link it takes me to one that says "Content Engine Feedback"

Also same for the Questions section `ce-beta@wpengine.com` might need to be acm-beta@wpengine if we can get the distro made.

Reply:
NEVERMIND. this is content engine.